### PR TITLE
feat: Add used slots inventory function and fix free slots

### PR DESCRIPTION
**Pull request by Rafii2198**
This is my first time tinkering with another mod, please let me know if I broke naming conventions or other stuff that could be improved!

### CODE REVIEW

File: common/src/main/java/com/wynntils/core/functions/FunctionManager.java [hunk 1]
```diff
@@ -306,6 +306,9 @@ public static void init() {
         registerFunction(new CharacterFunctions.HealthMaxFunction());
         registerFunction(new CharacterFunctions.HealthPctFunction());
         registerFunction(new CharacterFunctions.InventoryFreeFunction());
+        registerFunction(new CharacterFunctions.InventoryWithEquipmentFreeFunction());
+        registerFunction(new CharacterFunctions.InventoryUsedFunction());
+        registerFunction(new CharacterFunctions.InventoryWithEqipmentUsedFunction());
```
Comment (Incompleteusern) [inline common/src/main/java/com/wynntils/core/functions/FunctionManager.java]: Try to alphabetize

File: common/src/main/java/com/wynntils/wynn/model/PlayerInventoryModel.java [hunk 2]
```diff
@@ -55,12 +57,14 @@ public static void onSlotSetEvent(SetSlotEvent.Post e) {
     private static void updateCache() {
         InventoryMenu inventory = McUtils.inventoryMenu();
         emeralds = ContainerUtils.getEmeraldCountInContainer(inventory);
-        openSlots = ContainerUtils.getEmptySlotsInContainer(inventory);
+        openSlots = InventoryUtils.getEmptySlots(McUtils.inventory());
+        usedSlots = 28 - InventoryUtils.getEmptySlots(McUtils.inventory());
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/wynn/model/PlayerInventoryModel.java]: No need for this. Just returns 28 - openSlots in `getUsedInvSlots`

Comment (Rafii2198) [reply common/src/main/java/com/wynntils/wynn/model/PlayerInventoryModel.java]: Alright, done!

**Review by kristofbolyai — APPROVED** [review]
LGTM! Great work!

**Comment by Rafii2198** [comment]
Thank you for help and guidance!

**Pull request merged**